Dockerfile

```diff
@@ -153,6 +153,23 @@ RUN curl -ksSL https://github.com/CocoaPods/CocoaPods/archive/9461b346aeb8cba6df
 
 COPY docker/ruby.sh /etc/ort/bash_modules
 
+#------------------------------------------------------------------------
+# SCANCODE - Build scancode as a separate component
+FROM pythonbuild AS scancodebuild
+
+ARG SCANCODE_VERSION="3.2.1rc2"
```

Comment:
This is a step backwards. We use version 30.1.0 already in the current `Dockerfile`.